package.json

```diff
@@ -39,6 +39,7 @@
     "parent-require": "~1.0.0",
     "prismjs": "~1.15.0",
     "ps-node": "~0.1.6",
+    "reload": "git+https://github.com/alallier/reload.git#3.0.0",
```

Comment:
This needs to change, only here so that I can make sure we are set on integration before merging on the reload side

Reply:
still waiting on this

but is everything else done?

Reply:
Was waiting for final word on this end before merging reload.

Still need unit tests as well.

Reply:
This branch is published now. I will make the update

Reply:
I don't believe pointing this to git is necessary anymore